crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -490,15 +491,25 @@ fn declare(
     );
     let tx = DeclareTransaction::new(
         starknet_api::transaction::DeclareTransaction::V2(declare_tx),
+        // FIXME incorrectly calculating hashes may be the case of the problems
+        TransactionHash(StarkFelt::from(1_u32)),
         contract_class,
     )
     .unwrap_or_else(|err| panic!("Unable to build transaction {err:?}"));
 
     let account_tx = AccountTransaction::Declare(tx);
     let block_context = build_block_context();
-    let _tx_result = account_tx
-        .execute(blockifier_state, &block_context)
-        .context("Failed to execute declare transaction")?;
+    let tx_result = account_tx
+        // FIXME not sure we should be using true or false for these
```

Comment:
FIXME

Reply:
I guess we should at least agree on what should be used here and remove the fixme before the merge.

Reply:
Leaving it as `true`, `true` as it seems to make sense

#371